doc/nrf/ug_bootloader_external_flash.rst

```diff
@@ -3,13 +3,66 @@
 Using external flash memory partitions
 ######################################
 
+
 When using MCUboot, you can store the storage partition for the secondary slot in the external flash memory, using a driver for the external flash memory that supports the following features:
 
 * Single-byte read and write.
 * Writing data from the internal flash memory to the external flash memory.
 
+.. note::
+
+   The Partition Manager will only allow run-time access to flash partitions that are
+   placed on external flash devices that have drivers compiled in. Partition Manager has no
+   way to determine whether partitions are used at run-time or not, during compilation,
+   but it will decide to not include partition into map if there is no driver enabled for it.
```

Comment:
```suggestion
   way to determine whether partitions are used at run-time or not during compilation,
   but it will decide to not include partition into map if there is no driver enabled for it.
```